core/src/main/java/org/lflang/generator/c/CCmakeGenerator.java

```diff
@@ -153,6 +168,8 @@ CodeBuilder generateCMakeCode(
             + " gcc\")");
     cMakeCode.pr("  endif()");
     cMakeCode.pr("endif()");
+    // remove warnings for making building easier
+    cMakeCode.pr("set(CMAKE_C_FLAGS \"${CMAKE_C_FLAGS} -w\")");
```

Comment:
I have two concerns with this:
- I do not think it is portable: One big reason to use CMake is to allow to build with different compilers. `CMAKE_C_FLAGS` bypasses CMake to pass arguments directly to the underlying C compiler. Will this work with MSVC?
- Wrong code should look wrong (e.g. by producing warnings). I know that our code base produces warnings, but lots of effort has gone into reducing the number of warnings and probably more effort should continue to go into eliminating the warnings if we want the code to work reliably.

Reply:
This is fair but debugging application code with the warnings on is currently very difficult. I think it would be beneficial for students to not have to parse through them. I've moved the directive to only insert for the rp2040 platform which shouldn't mess with the portability. This is only a temporary fix for usability.